File: Ejercicio/c sharp/Binario/CantidadBinarios.cs
using System;

class ContarUnos {
    static void Main() {
        string binario = "10101101";
        int contador = 0;

        foreach (char c in binario) {
            if (c == '1') contador++;
        }

        Console.WriteLine($"El número de unos en {binario} es: {contador}");
    }
}

File: Ejercicio/GO/Binario/CantidadBinarios.go
package main

import "fmt"

func main() {
	binario := "10101101"
	contador := 0

	for _, c := range binario {
		if c == '1' {
			contador++
		}
	}

	fmt.Printf("El número de unos en %s es: %d\n", binario, contador)
}
